pkg/apis/batch/v1alpha1/job.go

```diff
@@ -174,6 +175,7 @@ type LifecyclePolicy struct {
 	Timeout *metav1.Duration `json:"timeout,omitempty" protobuf:"bytes,5,opt,name=timeout"`
 }
 
+// +kubebuilder:validation:Enum=none;best-effort;restricted;single-numa-node
```

Comment:
Can this take effect? Shouldn't it be added to `TopologyPolicy NumaPolicy `json:"topologyPolicy,omitempty" protobuf:"bytes,6,opt,name=topologyPolicy"``?

Reply:
Yes, it works. Added in struct, better for expense.
ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1/prometheus_types.go#L35-L51

Reply:
Great!